webtool/test_correct.py

```
@@ -1830,8 +1830,12 @@ def test_cmd_run_ueberlebt_ein_nicht_versucht(monkeypatch, tmp_path):
 # startet leer. Der zweite Test faehrt genau diesen Fall.
 
 
-def _sperr_projekt(monkeypatch, tmp_path, name="Sperr", welche=("S1",), mit_audio=False):
-    """Kleinstes Projekt, das `cmd_run` durchlaeuft — ohne Glossar- und Kontext-Aufruf."""
+def _sperr_projekt(monkeypatch, tmp_path, name="Sperr", welche=("S1",), mit_audio=False,
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_sperr_projekt`

(ANN202)